StitchCore/Sources/StitchCore/Internal/Net/Requests/Request.swift

```diff
@@ -96,30 +137,32 @@ public struct Request: Buildee {
      */
     public var body: Data?
 
-    // MARK: Initializer
-
-    /**
-     * Initializes this request by accepting a `RequestBuilder`.
-     *
-     * - throws: `RequestBuilderError` if the builder is missing an HTTP method, a URL, or a timeout.
-     */
-    public init(_ builder: RequestBuilder) throws {
-        guard let method = builder.method else {
-            throw RequestBuilderError.missingMethod
-        }
-        guard let url = builder.url else {
-            throw RequestBuilderError.missingUrl
-        }
-        guard let timeout = builder.timeout else {
-            throw RequestBuilderError.missingTimeout
-        }
-
+    // MARK: Initializers
+    
+    internal init(request: Request) {
+        self.method = request.method
+        self.url = request.url
+        self.timeout = request.timeout
+        self.headers = request.headers
+        self.body = request.body
+    }
+    
+    internal init(method: Method,
+                  url: String,
+                  timeout: TimeInterval,
+                  headers: [String: String],
+                  body: Data?) {
         self.method = method
         self.url = url
         self.timeout = timeout
+        self.headers = headers
+        self.body = body
+    }
 
-        self.headers = builder.headers ?? [:]
-        self.body = builder.body
+    public static func == (lhs: Request, rhs: Request) -> Bool {
```

Comment:
[style] extra space

Reply:
This seems to be how it's done everywhere else, but I'm making the change if this is the style we want to follow.